inferentia/qa/setup_test_enviroment_and_test.sh

```diff
@@ -131,11 +131,12 @@ if [ "${UPSTREAM_CONTAINER_VERSION}" = "" ]; then
     echo "found upstream container version: ${UPSTREAM_CONTAINER_VERSION} from build.py"
 fi
 
+set -e 
 # Build container with only python backend 
 cd ${TRITON_PATH}/server
 pip3 install docker
-./build.py --build-dir=/tmp/tritonbuild \
-           --cmake-dir=${TRITON_PATH}/server/build \
+./build.py -v \
```

Comment:
I would rather use the full name flag `--verbose` just to be more transparent on what is this for